app/api/contests/get_qualification_state.go

```diff
@@ -151,12 +160,13 @@ func (srv *Service) getContestInfoAndQualificationStateFromRequest(r *http.Reque
 		AllowsMultipleAttempts   bool
 		ContestMaxTeamSize       int32
 		ContestEnteringCondition string
+		EntryFrozenTeams         bool
```

Comment:
Not really clear what a bool called "EntryFrozenTeams" means.

Reply:
Actually the column name has the same problem...

Reply:
The fields of this stucture have names of the related db columns, also they are not part of the output. Do we still want to rename 'EntryFrozenTeams'?